core/rpc/src/chain/number.rs

```diff
@@ -41,7 +41,7 @@ impl<Number: traits::As<u64>> NumberOrHex<Number> {
 		match self {
 			NumberOrHex::Number(n) => Ok(n),
 			NumberOrHex::Hex(h) => {
-				// TODO [ToDr] this only supports `u64` since `BlockNumber` is `As<u64>` we could possibly go with `u128`. (#1377)
+				// FIXME [ToDr] this only supports `u64` since `BlockNumber` is `As<u64>` we could possibly go with `u128`. (#1377)
```

Comment:
```suggestion
				// FIXME this only supports `u64` since `BlockNumber` is `As<u64>` we could possibly go with `u128`. (#1377)
```

just clean up the comment a bit